RUFAS/biophysical/animal/animal_module_reporter.py

```diff
@@ -638,23 +638,77 @@ def report_enteric_methane_emission(cls, enteric_methane_emission_by_pen: dict[s
         }
         for pen_id_combination, enteric_methane_emission in enteric_methane_emission_by_pen.items():
             om.add_variable(
-                f"enteric_methane_emission_for_pen_{pen_id_combination}",
+                f"enteric_methane_emission_for_{pen_id_combination}",
                 enteric_methane_emission,
                 dict(info_map, **{"units": MeasurementUnits.GRAMS}),
             )
 
     @classmethod
-    def report_animal_module_manure(
-        cls,
-        manure_excretions_output_data: list[dict[str, PenManureData | dict[int, list[dict[str, ManureStream]]]]],
+    def report_manure_streams(cls, manure_streams: dict[str, ManureStream], simulation_day: int) -> None:
+        """
+        Generate detailed report of manure streams in the Animal Module.
+
+        Parameters
+        ----------
+        manure_streams : dict[str, ManureStream]
```

Comment:
```suggestion
        manure_streams : dict[str, ManureStream | dict[str, Any]]
```
I'm not sure what the dict is that would be received here and needs to be handled by the elif statement but should be accounted for here unless I'm misunderstanding.